src/ngraph/op/parameter.cpp

```diff
@@ -21,8 +21,11 @@
 using namespace std;
 using namespace ngraph;
 
-op::Parameter::Parameter(const element::Type& element_type, const Shape& shape)
+op::Parameter::Parameter(const element::Type& element_type,
+                         const Shape& shape,
+                         const bool memoizable)
     : Op("Parameter", {})
+    , m_memoizable(memoizable)
```

Comment:
Should be on `descriptor::Output`.
For example, in drop out, the value used for a weight is scaled by `p`, so if you update the weight, you would want to scale that by `p` and it would be the result of that multiply that would be going into the dot.

Reply:
This is a cosmetic flag and the only reason I have it there is to help with nbench (see Line 225 in benchmark.cpp). We dont do any optimizations in the graph based on this information currently. I can move it to output but I dont see how that is going to  help drop out. All the control information originates from runtime tensors and the graph data flow.

Reply:
In that case, just add it to parameter.rst

Reply:
will do.